styx-common/src/main/java/com/spotify/styx/util/ShardedCounterSnapshotFactory.java

```diff
@@ -82,12 +81,12 @@ private static void initShardRange(Storage storage, String counterId, int startI
                                      int endIndex) {
     try {
       storage.runInTransaction(tx -> {
-        IntStream.range(startIndex, endIndex).forEach(index -> {
+        for (int index = startIndex; index < endIndex; index++) {
```

Comment:
I like this old school better. It also gives space for compiler optimisation.

Reply:
Yeah, I felt like this was an egregious use of stream, but that's bonus. The real reason for this change is that `tx.store()` now throws `IOException`.